decaffeinate.sh

```diff
@@ -0,0 +1,53 @@
+set -ex
+
+curl -o .eslintrc https://raw.githubusercontent.com/sharelatex/web-sharelatex-internal/master/.eslintrc?token=AMHVk43L42-Htd0MSIfjao6vRukW52r4ks5cHA9WwA%3D%3D
```

Comment:
Maybe we should just pull these from https://github.com/sharelatex/web-sharelatex instead of using the tokens? They are open source.

Reply:
good point. im so used to using internal, i forgot about that.